tests/e2e/utils/ols_installer.py

```
@@ -316,6 +316,67 @@ def create_secrets(provider_name: str, creds: str, provider_size: int) -> None:
+            [  # noqa: S607
+                "operator-sdk",
+                "cleanup",
+                "lightspeed-operator",
+                "-n",
+                "openshift-lightspeed",
+            ],
+            capture_output=True,
+            text=True,
+            check=False,
+        )
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)